projects/netgrif-components-core/src/lib/view/task-view/service/public-task-loading.service.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { TestBed } from '@angular/core/testing';
+
+import {Injectable} from '@angular/core';
+import {LoadingEmitter} from '../../../utility/loading-emitter';
+import {PublicTaskLoadingService} from './public-task-loading.service';
+
+describe('PublicTaskLoadingService', () => {
+  let service: TestPublicTaskLoadingService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+        providers: [TestPublicTaskLoadingService]
+    });
+    service = TestBed.inject(TestPublicTaskLoadingService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should set value', () => {
+        service.setLoading$(true);
+        expect(service.getEmitter().isActive).toBeTrue();
+  });
+
+  it('should get value', () => {
+        const loading = service.getEmitter().asObservable();
+        expect(service.loading$).toEqual(loading);
+    });
```

Comment:
please add afterEach method to reset testing module, and fix intend

Reply:
done